src/handlers/miner_hbbft_handler.erl

```diff
@@ -297,6 +297,37 @@ handle_message(<<BinMsgIn/binary>>, Index, #state{hbbft = HBBFT, skip_votes = Sk
         {ok, {proposed_skip, ProposedRound, IndexRound}} ->
             Skips1 = Skips#{Index => {ProposedRound, IndexRound}},
             {S0#state{skip_votes = Skips1}, []};
+        {ok, {skip, Round, Sigs}} when Round /= CurRound ->
+            case lists:all(fun({SigIndex, Sig}) ->
+                                   libp2p_crypto:verify(t2b({proposed_skip, Round}), Sig, libp2p_crypto:bin_to_pubkey(lists:nth(SigIndex, S0#state.members)))
```

Comment:
we should recheck the length exceeds the threshold here too

Reply:
good call.

Reply:
Done.